### ISSUE TITLE
1 Blocked domain if Pihole is disabled

### ISSUE THREAD

**Issue by nicedevil007** [pi-hole/AdminLTE#1019]
Hey devs again,

I'm wondering a bit how this could be but, if I disable pihole for a given time it first shows me 0 blocked domains (because pihole is offline). Sometimes it shows me 1 blocked domain after a few seconds or minutes while it is still in offline/disabled status.

This is leading to an issue with the API. Because the status on the API seems to be connected to the number of Blocked Domains and not the status of pihole itself. So Pihole shows me "offline" and 1 blocked domain and the API tells me Pihole is enabled.

You can see this behaviour on the first screen posted in an issue @naimo84's node-red Node

https://github.com/naimo84/node-red-contrib-pi-hole-remote/issues/4

**Comment by DL6ER** [comment]
Based on this screenshot in the linked issue ticket,
![](https://user-images.githubusercontent.com/17103076/65975980-c0e13c00-e46f-11e9-8511-5b9ef656b8bf.png)
it doesn't seem like Pi-hole is still disabled. It would be good to know which domain is blocked, is there maybe only one entry in the blacklist on this client or one entry in the regex list, or ... ?

**Comment by nicedevil007** [comment]
I thought this as well and yes there is/was one page on the Blacklist as Exact Blocking and Regex / Wildcardblocking (same entry on everything).

I tested it while having this issue with 1 blocked Domain if PiHole is offline: Removed the Blacklist entry and then refreshed the browser and toggled PiHole as well. Unfurtunately it doesn't make any difference. But I'm realy sure that this is the domain that is leading to this issue because some days before there were 2 entries there and then PiHole told me that there are 2 Blocked Domains while still beeing disabled.
